Cargo.toml

```diff
@@ -14,23 +14,26 @@ documentation = "https://esp-rs.github.io/esp-idf-hal/"
 rust-version = "1.61"
 
 [features]
-default = ["std", "alloc", "esp-idf-sys"]
+default = ["std", "alloc", "esp-idf-sys", "async"]
 std = ["alloc", "esp-idf-sys/std", "edge-executor?/std"]
 alloc = []
 riscv-ulp-hal = []
+async = ["embedded-hal-async", "atomic-waker"]
 
 [dependencies]
 nb = "1.0.0"
 embedded-can = "0.4.1"
 embedded-hal = "=1.0.0-alpha.9"
-embedded-hal-0-2 = { package = "embedded-hal", version = "0.2.7", features = ["unproven"] }
+embedded-hal-0-2 = { package = "embedded-hal", version = "0.2.7", features = [ "unproven", ] }
```

Comment:
Why the comma?

Reply:
It got autoformatted by my LSP setup and i tried to fix it, but apparently missed a spot. I'll fix.